sources/multi.suwayomi/res/source.json

```
@@ -0,0 +1,17 @@
+{
+	"info": {
+		"id": "mulit.suwayomi",
```

oops, I didn't see this before
```suggestion
		"id": "multi.suwayomi",
```